### PR TITLE
Javascript question about data type

### PR DESCRIPTION
**Pull request by ddoyediran**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by ddoyediran** [comment]
@jdwilkin4 please, let me know if there is anything you want me to remove or add. Thank you

File: src/data/javascript-quiz.ts [hunk 1]
```diff
@@ -457,6 +457,17 @@ const javascriptQuiz = [
...
       "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Addition",
   },
+  {
+    Question: "What are the two basic groups of data types in JavaScript?",
+    Answer: "Primitive and Reference types",
+    Distractor1: "Enumerated and Object types",
+    Distractor2: "Primitive and Floating types",
+    Distractor3: "Character and Number types",
+    Explanation:
+      "Primitive and Reference types are the two basic groups of data types in Javascript.",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:467]:
```suggestion
      "Primitive and Reference types are the two basic groups of data types in JavaScript.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by ddoyediran** [comment]
Done. Thank you

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**